code/datums/wounds/cranial_fissure.dm

```diff
@@ -0,0 +1,120 @@
+/datum/wound_pregen_data/cranial_fissure
+	wound_path_to_generate = /datum/wound/cranial_fissure
+	required_limb_biostate = BIO_BONE
+
+	required_wounding_types = list(WOUND_ALL)
+
+	wound_series = WOUND_SERIES_CRANIAL_FISSURE
+
+	threshold_minimum = 150
+	weight = 10
+
+	viable_zones = list(BODY_ZONE_HEAD)
+
+/datum/wound_pregen_data/cranial_fissure/get_weight(obj/item/bodypart/limb, woundtype, damage, attack_direction, damage_source)
+	if (limb.owner?.stat < HARD_CRIT)
+		return 0
+
+	return ..()
+
+/datum/wound/cranial_fissure
+	name = "Cranial Fissure"
+	desc = "Patient's crown is agape, revealing severe damage to the skull."
+	treat_text = "Immediate surgical reconstruction of the skull."
+	examine_desc = "is split open"
+	occur_text = "is split into two separated chunks"
+
+	simple_desc = "Patient's skull is split open."
+	threshold_penalty = 40
+
+	severity = WOUND_SEVERITY_CRITICAL
+	sound_effect = 'sound/effects/dismember.ogg'
+
+#define CRANIAL_FISSURE_FILTER_DISPLACEMENT "cranial_fissure_displacement"
+
+/datum/wound/cranial_fissure/wound_injury(datum/wound/old_wound = null, attack_direction = null)
+	ADD_TRAIT(limb, TRAIT_IMMUNE_TO_CRANIAL_FISSURE, type)
+	ADD_TRAIT(victim, TRAIT_HAS_CRANIAL_FISSURE, type)
+
+	victim.add_filter(CRANIAL_FISSURE_FILTER_DISPLACEMENT, 2, displacement_map_filter(icon('icons/effects/cranial_fissure.dmi', "displacement"), size = 3))
```

Comment:
How does this interact with monkeys or spacers? Or hats and glasses? Looks like this doesn't modifiy the head, but the final mob sprite and just assumes the head is always in the same place (it's not ;_;)

You might want to consider applying it to the head bodypart and blocking headgear

Adding your filter to the output of head/get_limb_icon will do this

Reply:
I tried making it work with the head bodypart but the rendering code very much expects to be collecting into an overlays list at the end. I tried the exact description you were talking about, but it doesn't work because get_list_icon is returning a list for overlays. I tried combining them into one and then putting that on a mutable_appearance, but that also just didn't wrok.

Reply:
Yeah okay, that's the end of that then. I suppose it's fine for now